pkg/k8s/clusterresourcefactory.go

```diff
@@ -41,7 +42,7 @@ func (factory *ClusterResourceFactoryOptions) AttachCobraCliFlags(cmd *cobra.Com
 		"The namespace to keep AWS accounts. The default value is aws-account-operator.")
 	cmd.Flags().StringVarP(&factory.AccountName, "account-name", "a", "", "The AWS account CR we need to create a temporary AWS console URL for")
 	cmd.Flags().StringVarP(&factory.AccountID, "account-id", "i", "", "The AWS account ID we need to create AWS credentials for -- This argument will not work for CCS accounts")
-	cmd.Flags().StringVarP(&factory.ClusterID, "cluster-id", "C", "", "The Internal Cluster ID from Hive to create AWS console URL for")
+	cmd.Flags().StringVarP(&factory.ClusterID, "cluster-identifier", "C", "", "The Internal Cluster ID/External Cluster ID/ Cluster Name from Hive to create AWS console URL for")
```

Comment:
Actually, I think we need to add an alias here if we're going to change from `cluster-id` to `cluster-identifier` as that would constitute a breaking change and could break any scripts that use this flag.

Reply:
Hey Kirk! would it be ok if i can leave the cluster-id as is ?